src/serde_multipart/mod.rs

```diff
@@ -31,7 +31,6 @@ pub(crate) fn to_form<T: ?Sized + Serialize>(val: &T) -> impl Future<Output = Re
 }
 
 // https://github.com/teloxide/teloxide/issues/473
-#[cfg(test)]
```

Comment:
`cfg(test)` is actually needed here though https://github.com/teloxide/teloxide-core/pull/159#discussion_r777228274